optuna/study.py

```diff
@@ -33,6 +33,11 @@ class Study(object):
     This object provides interfaces to run a new :class:`~optuna.trial.Trial`, access trials'
     history, set/get user-defined attributes of the study itself.
 
+    For creating and loading studies, please refer to the documentation of
+    :func:`~optuna.study.create_study` and :func:`~optuna.study.load_study` respectively.
+
+    Note that the direct use of this constructor is not recommended.
```

Comment:
In my opinion, this sentence had better be presented in an upper line. E.g., just after `A study corresponds to an optimization task, i.e., a set of trials.`

Reply:
Sounds good. I will move it.

Reply:
Moved: https://github.com/pfnet/optuna/pull/330/commits/b0cfea8d5caed1bb25ab54717260ce73f118c47c